### PR TITLE
Fix pagination template styling

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/15503

This PR fixes the pagination styling for r&s templates.

## Testing done
N/A

## Screenshots
image

## Acceptance criteria
- [x] Update pagination styling for r&s templates.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/includes/pagination.drupal.liquid [hunk 1]
```diff
@@ -31,13 +31,17 @@
...
+          {% if page.class %}
+            class="{{ page.class }}"
+          {% endif %}
+          {% if page.class != "va-pagination-active" %}
+            href="{{ page.href }}"
+          {% endif %}
+        >
           {{ page.label }}
-        {% endif %}
+        </a>
```
Comment (ncksllvn) [inline src/site/includes/pagination.drupal.liquid:44]: Ohhh, good fix. You think it was just a logical error in the template before? I didn't understand what it was trying to do.

**Review by ncksllvn — APPROVED** [review]
Great job, love little fixes like these that have a great effect

File: src/site/includes/pagination.drupal.liquid (hunk 1 above)
Comment (kelsonic) [reply src/site/includes/pagination.drupal.liquid:44]: Yupp it was! 🙂

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**